addons/project_account/models/project_project.py

```diff
@@ -139,7 +140,13 @@ def action_profitability_items(self, section_name, domain=None, res_id=False):
     def _get_domain_aal_with_no_move_line(self):
         """ this method is used in order to overwrite the domain in sale_timesheet module. Since the field 'project_id' is added to the "analytic line" model
         in the hr_timesheet module, we can't add the condition ('project_id', '=', False) here. """
-        return [('account_id', '=', self.analytic_account_id.id), ('move_line_id', '=', False), ('category', '!=', 'manufacturing_order')]
+        plans = self.env['account.analytic.plan']
+        plans = sum(plans._get_all_plans(), plans)
+        line_columns = [p._column_name() for p in plans]
+
+        # The project account column depends on the analytic account's plan_id
+        account_domain = expression.OR([[(col, '=', self.analytic_account_id.id)] for col in line_columns])
```

Comment:
cannot we use `auto_account_id` field instead of building such domain to check in all plans? 

It seems to me the search method of `auto_account_id` will do the same than we did here, right? 🙂